includes/CacheTypes/Browser.php

```diff
@@ -100,17 +100,16 @@ public static function addRules( $cacheLevel ) {
 		}
 		$rules[] = '</IfModule>';
 
-		$cache_exclusion_parameters = array_map( 'trim', explode( ',', get_option( CacheExclusion::OPTION_CACHE_EXCLUSION ) ) );
+		$cache_exclusion_parameters = array_map( 'trim', explode( ',', sanitize_text_field( get_option( CacheExclusion::OPTION_CACHE_EXCLUSION ) ) ) );
```

Comment:
We should do some validation to make sure `get_option` returns a string else `sanitize_text_field` might throw a fatal error.